### PR TITLE
[PHP] Added missing keywords (trait, callable, insteadof, string, int)

### PR DESCRIPTION
**Pull request by ptcong**
- Added missing keywords (trait, callable, insteadof)

trait keyword
```
trait Taggable
{
}
```

improve use meta.

Now `as` keywords not being `operator` keyword

Added `insteadof` keyword

```
class Xyz {
    use A, B, C {
        C:method2 insteadof B;
        B::method1 as anotherOne1;
        B::method2 as anotherOne2;
    }
}

```

Added `callable`, `int`, `string` type hint
```
public function abc(callable $var, int $var2, string $var3)
{
   
}
```

### CODE REVIEW

**Comment by wbond** [comment]
We’ll need tests for these additions. Check out http://www.sublimetext.com/docs/3/syntax.html#Testing for more info.

**Comment by ptcong** [comment]
@wbond ok, i just made the tests

**Comment by ptcong** [comment]
@wbond recheck the PR, i just add the tests

File: PHP/PHP Source.sublime-syntax [hunk 1]
```diff
@@ -373,29 +373,54 @@ contexts:
         1: keyword.other.use.php
       push:
         - meta_scope: meta.use.php
-        - match: (?=;|(?:^\s*$))
+        - match: (?=;|(?:\s*$)|\s+)
           pop: true
```
Comment (wbond) [inline PHP/PHP Source.sublime-syntax]: This doesn't seem right. Does PHP really end a use statement as soon as a one or more spaces is present?

Comment (ptcong) [inline PHP/PHP Source.sublime-syntax]: Just for pop from
```
use Trait1, Trait2 {
    Trait1:methodA as methodX;
} [this is space]
protected $prop1;
```

**Comment by ptcong** [comment]
@wbond i just remke this commit, it seems works fine now.

**Comment by michaelblyons** [comment]
Can you prefix your pull request title with [PHP]?

**Comment by ptcong** [comment]
@michaelblyons ok i just added [PHP] prefix for this PR

File: PHP/PHP Source.sublime-syntax [hunk 2]
```diff
@@ -106,7 +106,7 @@ contexts:
     - include: comments
     - match: |-
         (?x)
-        \s*(array) # Typehint
+        \s*(array|callable|int|string) # Typehint
```
Comment (duncan3dc) [inline PHP/PHP Source.sublime-syntax]: What about `bool` and `float`? (http://php.net/manual/en/functions.arguments.php#functions.arguments.type-declaration)

**Comment by wbond** [comment]
I rebased after merging these in, so the hashes changed. Here are the new commits:

4397c636
004c616a
55be38a1

**Pull request closed without merging**